app/views/layouts/_doctype.html.haml

```diff
@@ -0,0 +1 @@
+!!! 5
```

Comment:
@martinpovolny I will not hold up this PR for implementing my feedback.  But would like it done in a future PR because a one-line file with `!!! 5` will force someone to look up what the heck it means.